packages/react/src/components/Authenticator/styles.css

```diff
@@ -63,3 +63,46 @@
 [data-amplify-authenticator] :disabled {
   @apply opacity-50;
 }
+
+:root {
+  /* primary button colors */
+  --amplify-components-button-primary-background-color: #7256ff;
+  --amplify-components-button-primary-border-color: #7256ff;
```

Comment:
@dbanksdesign This change would mean that these default values would be overridden for all components for anyone importing the Authenticator.

Should we scope these to `[data-amplify-authenticator]` or re-evaluate the default values of these variables?

Reply:
Yeah, let me know what would be the correct way to go. I went with `:root` because that's what the primitives docs say - https://ui.docs.amplify.aws/ui/primitives/button .

Reply:
Updated in new commit to use `data-amplify-authenticator`